components/menu/OverflowMenu/style.module.scss

```diff
@@ -6,36 +6,47 @@
 }
 
 .overflow-menu__button {
-  @include circle(26px);
+  @include circle(36px);
   padding: $size-md;
   border: none;
   background-color: $color-gray;
   display: inline-block;
   cursor: pointer;
   position: relative;
+  box-shadow: inset 0 1px 0 0 rgba(0, 0, 0, 0.08);
 
   &:focus {
     outline: 0;
   }
 }
 
-.overflow-menu__svg {
+.overflow-menu__svg,
+.overflow-menu__svg--opened {
   @include completely-center();
-  width: 40%;
-  height: 60%;
+  width: 36.111111111%;
+  height: 44.444444444%;
+}
+
+.overflow-menu__svg {
+  fill: $color-deepgrey;
+}
+
+.overflow-menu__svg--opened {
+  fill: $color-m-primary-blue;
```

Comment:
Legacy color variable

Reply:
Is there a different one I should be using? I do see a different one of $color-blue.